### ISSUE TITLE
Is it possible to train Casanovo using EThcD spectra?

### ISSUE THREAD

**Issue by DIGEdige** [Noble-Lab/casanovo#424]
I have over 20,000 EThcD spectra as training data, and I used Casanovo to train a new model (with max_epochs = 100, val_check_interval = 50000, nbeams = 10, and n_peaks = 150). However, on a test set of over 10,000 EThcD spectra, the peptide recall was only 0.6%.

With the same amount of data and parameter settings, the peptide recall reached over 20% when using HCD spectra. I would like to ask: how can I modify my training or prediction strategies to make Casanovo achieve sequencing accuracy comparable to that of HCD spectra?

**Comment by wsnoble** [comment]
Though we have not tried training Casanovo on this kind of data, I don't see any reason why it shouldn't be able to learn from EThcD spectra.  If you want to share your training and validation set, we can see if we can reproduce your results and potentially suggest alternative training strategies.

You can email the files wnoble@uw.edu if they are too large to upload to github.

**Comment by bittremieux** [comment]
Can you include some additional details, such as the Casanovo version you're using and the log produced during training, to get some more information whether everything is working correctly?

Although if you've got a similar set-up working for HCD data, I agree with Bill that there shouldn't be anything special about EThcD data that should make training fail in that setting.

Note though that you're using a relatively small training dataset, which could result in two issues:
1. The dataset is too small to learn relevant patterns for more complex EThcD spectra.
2. You're entirely overfitting to a small training dataset, with 100 epochs, and performance doesn't generalize to the test set.

The training and validation log produced during training might give us some more insights into whether any of these issues are occurring.

**Comment by bittremieux** [comment]
Closing due to insufficient details. Feel free to reopen this issue in case you have further questions.
